btr-api/src/btr_api/resources/submission.py

```diff
@@ -172,6 +166,25 @@ def create_register():
             current_app.logger.info(err.error)
             current_app.logger.error('Error updating search record for submission: %s', submission.id)
 
+        try:
+            # NOTE: will be moved to person or ownership table @event.listens_for after_insert #23291
```

Comment:
Why not make it in services, and just call it in the resource, or from wherever needs to move to ?

Reply:
Eh we could do that I guess, but I've been trying to stay away from extrapolating model code out into services unless there's a lot of code / operations in there. It's less jumping around to follow the code and we'll have pretty tiny models

Reply:
I guess for this piece it makes sense though since we've got a lot of other things its using. I will add it to a service like you're suggesting in my current ticket 23291